CONTRIBUTING.md

```diff
@@ -0,0 +1,39 @@
+# Contributing to Dapr Rust SDK
```

Comment:
Coudl you add info from https://github.com/dapr/dapr/blob/master/CONTRIBUTING.md to it.